addons/main/functions/fnc_handleHealEh.sqf

```diff
@@ -3,7 +3,7 @@
     params ["_unit"];
     (damage _unit) isEqualTo 0 || {!alive _unit}
 },{
-    params ["_unit", "_healer"];
+    params ["_unit", "_healer", "", "_consumeMedkit"];
```

Comment:
third param never used, what's up with that?

Reply:
also use default values for the additional params

Reply:
I was basing it off _this for `HandleHeal` which has a third value which is always false